src/main/java/org/team1540/robot2024/Robot.java

```diff
@@ -49,7 +60,7 @@ public void robotInit() {
         switch (Constants.currentMode) {
             case REAL:
                 // Running on a real robot, log to a USB stick ("/U/logs")
-                Logger.addDataReceiver(new WPILOGWriter());
+//                Logger.addDataReceiver(new WPILOGWriter());
```

Comment:
Maybe make this based on if the FMS is connected

Reply:
it doesn't know on robotInit though